### ISSUE TITLE
Error loading translation RangeError

### ISSUE THREAD

**Issue by vasilich6107** [ilteoood/flutter_i18n#65]
The error occurs here

```
Future<Locale> _findCurrentLocale() async {
    final String systemLocale = await findSystemLocale();
    MessagePrinter.info("The system locale is $systemLocale");
    final List<String> systemLocaleSplitted = systemLocale.split("_");
    final int countryCodeIndex = systemLocaleSplitted.length == 3 ? 2 : 1;
    return Future(() => Locale(
        systemLocaleSplitted[0], systemLocaleSplitted[countryCodeIndex]));
  }
```

On line 
```
return Future(() => Locale(
        systemLocaleSplitted[0], systemLocaleSplitted[countryCodeIndex]));
```

Console logs

```
flutter: [flutter_i18n INFO]: The system locale is en
flutter: [flutter_i18n DEBUG]: Error loading translation RangeError (index): Invalid value: Only valid value is 0: 1
```

**Comment by vasilich6107** [comment]
It seems that on IPad on iOs 13.3 the system locale returns without country code)))

**Comment by ilteoood** [comment]
Hi @vasilich6107,
we have a thread about iOS problems, you can find it there: https://github.com/ilteoood/flutter_i18n/issues/58.
Follow the instructions provided there, and please give me you feedback.
Thank you,
Matteo

**Comment by ilteoood** [comment]
This has been fixed with version 0.10.1

**Comment by vasilich6107** [comment]
It seems that this bug relates to the iOS 13.3 iPad emulators. On cold start the system locale is set to the English(English) this is why I saw the `en` in platform locale.

After you switch the locale to English(United States) or any other English(Country) you’ll never find English(English) again in the available locale list.

This strange bug costed me several hours)
